### PR TITLE
Feat : Added P1-Snackbar Component

### PR DESCRIPTION
**Pull request by TusharJade**
### Added P1-Snackbar Component
Screenshot (48)

### CODE REVIEW

File: Components/Snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+.snackbar-success{
+    background-color:var(--success-snackbar);
+    padding: 1rem;
+    border-radius: 0.3rem;
+    display: flex;
+    width: 27rem;
+    justify-content: space-between;
+    align-items: center;
+}
+
```
Comment (Avin008) [inline Components/Snackbar/snackbar.css:38]: remove blank space

**Review by Avin008 — APPROVED** [review]
good to merge

File: Components/Snackbar/snackbar.css (hunk 1 above)
Comment (SumitSaundadkar) [reply Components/Snackbar/snackbar.css:38]: Good to go.

Comment (TusharJade) [reply Components/Snackbar/snackbar.css:38]: Yup will remove blank spaces

**Review by SumitSaundadkar — APPROVED** [review]
code looks good just avoid extra spacing and name convention for proper understanding, just use _ for the class name instead -.everything looks good to me great work.

**Pull request merged**